### PR TITLE
added directory scan script 

### PR DESCRIPTION
**Pull request by sayanmondal2098**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        res.write('Hello Server');
+        res.end();    
+    }
+//it'll return all the file name in the directory
+    if(req.url==='/dirname'){
+        res.write((path.dirname(__filename)));
+        res.write(directories);
+        console.log(dirlister);
+        res.end();
+    }
```
Comment (paulodiovani) [inline src/App.js:22]: We don't want to list the directory.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+});
+
+server.on('connection',()=>{
+    console.log("New Connection.....")
+});
```
Comment (paulodiovani) [inline src/App.js:28]: Probably not required...
But we could use for logs,

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+var path = require('path');
+var dirlister = require('./GetDirectory');
+
+// var directories = path.dirname('/Users/Refsnes/demo_path.js');
+// console.log(directories);
+
+
+// console.log(dirlister)
+// json = dirlister.JSON
+const server = http.createServer((req,res)=>{
```
Comment (paulodiovani) [inline src/App.js:11]: What about using `express` instead?

**Review by paulodiovani — CHANGES_REQUESTED** [review]
I suppose this is intended for the server bootstrap, but it is still too crude IMO.
I'll wait for improvements.

File: src/App.js (hunk 2 above)
Comment (sayanmondal2098) [reply src/App.js:28]: yes, for log we can use it

File: src/App.js (hunk 3 above)
Comment (sayanmondal2098) [reply src/App.js:11]: no problem , i'll try to modify it using express

**Pull request closed without merging**